plugins/mbaas/README.md

```diff
@@ -0,0 +1,4 @@
+
+Contents in here will overlap on top of the files from the default subtree
+
+*Note: Any files with the same sub path and name as a file from default will be overwritten with the one here (This allows for core/mbaas to override the behavior we have in default)*
```

Comment:
So this is very similar to what we do with our current SaaS offering and chef.  We have "default" files that can be overriden by ones based on os families and such.  Its unlikely we'll use the overriding concept here, its more so that we can have checks for just the core in particular that won't have any concept in the mbaas.  So rather than have the scripts that are purely core specific getting copied into a nagios pod for mbaas, it can avoid them entirely.  When the plugins/core path didn't exist and I removed the * (which was flatting the files/subfolders), it caused the ADD/COPY to fail because the source didn't exist.  Ironcially, I can remove the README's now since the folder will still stay around since I will have a commit that adds them and a commit that removes them.  I'm happy to take these README's back out.